framework/xds_k8s_flags.py

```diff
@@ -33,6 +33,18 @@
     help="Traffic Director gRPC Bootstrap Docker image",
 )
 
+CLOUDRUN_SERVER_IMAGE = flags.DEFINE_string(
```

Comment:
Why do we need separate images? I thought we'll be using our regular test server / test client.